server/db/migrations/20230309195953_update-campaign-model.js

```diff
@@ -0,0 +1,20 @@
+const tableName = 'campaigns'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.alterTable(tableName, (table) => {
+      // Auto-incrementing non-nullable unsigned integer primary key "id" field
+      table.increments()
+
+      // Simple fields
+      table.string('campaign_tagline').notNullable()
```

Comment:
I think these can be nullable, because we currently only need them for single-campaign mode.